Products/ZenRRD/zentestcommand.py

```diff
@@ -16,26 +16,21 @@
 
 __version__ = "$Revision$"[11:-2]
 
-import os
-import popen2
-import fcntl
-import time
-import sys
-import select
-import logging
-import signal
 from copy import copy
-log = logging.getLogger("zen.zentestcommand")
+import logging
+import sys
 
-import Globals
-from Products.ZenUtils.ZenScriptBase import ZenScriptBase
-from Products.DataCollector.SshClient import SshClient
+import Globals # noqa
+from Products.ZenUtils import Utils
+from Products.ZenUtils import ZenScriptBase
 
+
+log = logging.getLogger("zen.zentestcommand")
 snmptemplate = ("snmpwalk -c%(zSnmpCommunity)s "
                 "-%(zSnmpVer)s %(manageIp)s %(oid)s")
 
 
-class TestRunner(ZenScriptBase):
+class TestRunner(ZenScriptBase.ZenScriptBase):
 
     def __init__(self):
         ZenScriptBase.__init__(self, connect=True)
```

Comment:
So this piece of code is referring to module `ZenScriptBase` instead of the class with very same name? :confused: